outbound.js

```diff
@@ -352,7 +352,10 @@ exports.send_email = function () {
     var from = arguments[0],
         to   = arguments[1],
         contents = arguments[2];
-        var next = arguments[3];
+    var next = arguments[3];
+    var options = arguments[4];
+
+    var dot_stuffing = ((options && typeof(options.dot_stuffing) !== 'undefined') ? options.dot_stuffing : true);
```

Comment:
This needs to be documented.

I would be tempted to default this on, as we should probably presume that 'contents' isn't dot-stuffed when passed and the option would disable this functionality rather than enable it.

@baudehlo: Any thoughts on this?